### ISSUE TITLE
[HOLD App PR #39094][Wave Collect][QBO] Use `WORKSPACE_ACCOUNTING`'s `getRoute` when specifying the callback path for the QBO auth flow

### ISSUE THREAD

**Issue by hayata-suenaga** [Expensify/App#39216]
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~010f098ac205edac04</li>
        <li>Upwork Job ID: 1779907490337906688</li>
        <li>Last Price Increase: 2024-04-15</li>
    </ul>
</details>

**Comment by alexpensify** [comment]
Noted to follow the GHS listed in the title.

**Comment by trjExpensify** [comment]
Making sure this is captured in release 1.

**Comment by alexpensify** [comment]
Weekly Update: 

https://github.com/Expensify/App/pull/38283 has been deployed to staging 
https://github.com/Expensify/App/issues/39094 is in a draft PR

**Comment by hayata-suenaga** [comment]
> Weekly Update:
> https://github.com/Expensify/App/pull/38283 has been deployed to staging

A correction on the status of https://github.com/Expensify/App/pull/38283 PR: It hasn't been merged yet 🙇

**Comment by alexpensify** [comment]
Weekly Update: On hold and waiting on the other GHs.

**Comment by trjExpensify** [comment]
The (new) PR for implementing the accounting page is done, so we're down to waiting on [#39094](https://github.com/Expensify/App/issues/39094) for authentication which is deep in review. @hayata-suenaga can you confirm if this is internal or external? The OP is a little... light. 😅 Thanks!

**Comment by hayata-suenaga** [comment]
Hello Tom thank you for keeping an eye on this one 😄 This issue is rather a memo I'm using to make sure that a certain code is used in the implementation. We might not have to do this depending on the outcome of the linked PR, but even if we have to do this, this is just one line change 🙇

**Comment by trjExpensify** [comment]
Sounds good!

**Comment by trjExpensify** [comment]
The PR for #39094 has been deployed to staging. Can we make an assessment on this one now, so we either close it or take it off hold? Thanks!

**Comment by hayata-suenaga** [comment]
The PR uses the environment URL

```
onPress={() => Link.openLink(getQuickBooksOnlineSetupLink(policyID), environmentURL, false)}
```

we don't need to go ahead with this issue. Closing this now
